### ISSUE TITLE
Add accessibility for screen readers

### ISSUE THREAD

**Issue by fourcolors** [harvesthq/chosen#1446]
Many institutions need their dropdowns accessible to screen readers for people that can't see. There should be approprate aria labels and focus points so one can navigate the dropdown select with their keyboard and other software programs like JAWS/Voice Over. 

Here is a reference to the aria roles/labels 

http://www.w3.org/TR/wai-aria/states_and_properties

I'm going to be working on adding this today. I don't have windows with JAWs but if there is anyone that does, it would be nice to get help testing it.
